content/docs/tasks/policy-enforcement/rate-limiting/index.md

```diff
@@ -244,18 +99,8 @@ so the configuration to enable rate limiting on both adapters is the same.
 
 1. Confirm the `quota instance` was created:
 
-    {{< text bash yaml >}}
-    $ kubectl -n istio-system get quotas requestcount -o yaml
-    apiVersion: config.istio.io/v1alpha2
-    kind: quota
-    metadata:
-      name: requestcount
-      namespace: istio-system
-    spec:
-      dimensions:
-        source: request.headers["x-forwarded-for"] | "unknown"
-        destination: destination.labels["app"] | destination.service.host | "unknown"
-        destinationVersion: destination.labels["version"] | "unknown"
+    {{< text bash >}}
+    $ kubectl -n istio-system get instance requestcountquota -o yaml
     {{< /text >}}
```

Comment:
While on that, could you fix the following Paragraph, which mentions `destination.service.host`. The yaml file uses `destination.service.name`.

Reply:
Yes, will do.